synapse/lib/parser.py

```diff
@@ -723,7 +723,12 @@ def unescape(valu):
     The full list of escaped characters can be found at
     https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals
     '''
-    ret = ast.literal_eval(valu)
+    try:
+        ret = ast.literal_eval(valu)
+    except ValueError as e:
+        mesg = f"Invalid character in string {valu}: {e}"
```

Comment:
If we use the `repr(valu)` we don't drag around the raw invalid value in a printable string.

Reply:
updated to repr the value